lib/get-random-id.js

```diff
@@ -1,3 +1,3 @@
 module.exports = function() {
-  return require('sdk/util/uuid').uuid().number.replace('{','').replace('}','');
+  return require('sdk/util/uuid').uuid().number.replace('{', '').replace('}', '');
```

Comment:
I imagine so. The added spaces are just around the comma formatting, not around any strings or anything particularly interesting/useful.